.github/workflows/test.yml

```
@@ -66,8 +92,9 @@ jobs:
         run: yarn start -- --gen-schema && yarn gql-tada generate output
 
       - name: E2E Tests
-        run: yarn test:e2e --shard=${{ matrix.shard }}/6 --reporters=github-actions
+        run: yarn test:e2e --shard=${{ matrix.shard }}/6 --reporters=github-actions ${{ matrix.test-args }}
         env:
           NEO4J_VERSION: ${{ matrix.neo4j-version }}
```

actionlint (1.7.12)

[error] property "neo4j-version" is not defined in object type {database: string; shard: number; test-args: string}

(expression)